taichi/struct/struct_llvm.cpp

```diff
@@ -25,8 +25,8 @@ StructCompilerLLVM::StructCompilerLLVM(Program *prog, Arch arch)
 void StructCompilerLLVM::generate_types(SNode &snode) {
   TI_AUTO_PROF;
   auto type = snode.type;
-  if (snode.is_bit_level)
-    return;
+//  if (snode.is_bit_level)
+//    return;
```

Comment:
According to my understanding, we might also need to execute this function for bit-level SNodes.

Reply:
Actually, I don't think we should comment out these lines. Note that for `bit_level` SNodes we never actually generate their LLVM types (since LLVM doesn't really have corresponding types).

Reply:
Yes, I didn't comment them out at beginning, but the program will crash when visiting `SNodeLookUpStmt`, and I thought it might because we don't generate the types for bit-level SNodes.